scripts/openshiftci-presubmit-integratione2ebenchmark.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
the name of this file is horribly long.
How about `openshiftci-presubmit-test.sh ` or `openshiftci-presubmit-all-tests.sh`?

Reply:
I was expecting this comment :D

Renamed to ```openshiftci-presubmit-all-tests.sh```